clients/drcachesim/common/options.h

```diff
@@ -49,6 +49,7 @@
 #define SYSCALL_MIX "syscall_mix"
 #define VIEW "view"
 #define FUNC_VIEW "func_view"
+#define RECORD_VIEW "record_view"
```

Comment:
I see the docs listing the supported tools for -simulator_type were not updated (nor the .dox files listing example usage of all the tools) as it normally would be for a new tool: which may well be the right decision as I do not think this tool should be publicized or documented as we really do not want external users operating on trace_entry_t.  But please make that clear with a comment here as otherwise it looks like a mistake.

Reply:
Not adding it to the .dox was an oversight (but, I guess, a welcome one).
Since we want to limit the visibility of this tool, I agree on not adding it.

Reply:
As noted in the other thread, please rename to `internal_record_review`.

Reply:
Yes, done.